exercises/practice/high-scores/create_fixture.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE IF EXISTS "scores";
+CREATE TABLE "scores" (
+    "uuid" TEXT,
```

Comment:
`uuid` implies each score has a unique ID. Should this be `game_id` or suchlike?